hack/make-rules/test-e2e-node.sh

```diff
@@ -148,7 +148,7 @@ else
 
   # Do not use any network plugin by default. User could override the flags with
   # test_args.
-  test_args='--kubelet-flags="--network-plugin= --network-plugin-dir=" '$test_args
+  test_args='--kubelet-flags="--network-plugin= --cni-bin-dir=" '$test_args
```

Comment:
I think this has no effect on the tests. If it really has error, I will fix it in another PR, no need to block this PR.